site/content/demo/index.md

```diff
@@ -1,9 +1,11 @@
 ---
-layout: unstyled
+layout: docs
 ---
 
-`(layout/unstyled.js)`
-
-Demo page with an unstyled layout *eg. used for homepage with sections*
-
-<DemoComponent data={testData} />
\ No newline at end of file
+# Demos of some of our cool features
```

Comment:
Blank line after heading.

See https://playbook.datopian.com/style-guide/markdown/ for style guide for markdown